format_values.md

```diff
@@ -0,0 +1,90 @@
+# Format Values
+
+This document is a list of standardized values that can be used in certain fields in the format.
+
+## Frameworks
+
+The following is a list of standardized Framework names.
+
+- `framework-fabric-loader`
+- `framework-forge`
+- `framework-liteloader`
+- `framework-rift`
+- `framework-risugami-modloader`
+
+These names are prefixed with `framework-` to differentiate them between normal Projects. It also prevents possible collisions between Frameworks and other projects that have the same name.
+
+Hosts that serve Framework metadata MUST use these IDs. When a Framework ID is used in metadata, such as listed as a dependency, it MUST be one of the standardized values.
+
+---
+
+## Tags
+
+This is a list of standardized values that can be present in the `tags` field. It is not required to use these values, although it is encouraged to use them.
+
+- `mod`
+- `modpack`
+- `resourcepack`
+- `texturepack` (note: use `resourcepack` for packs that support it)
+- `datapack`
+- `world`
+- `framework`
+
+---
+
+## File Relations
+
+Files can have the following `rel` values:
+
+- `primary` - This is the primary file of this version, and what launchers are most likely to use
+- `source` - Source code of this version
+- `deobf` - Deobfuscation file for this version
+- `javadoc` - Downloadable Javadoc
+- `installerWin` - This is a generic installer for Windows
+- `installerMacos` - This is a generic installer for macOS
+- `installerLinux` - This is a generic installer for Linux
+- `installerGeneric` - A generic installer
```

Comment:
How should installers be executed?

Reply:
Also what's the separation between this and installation info? "installerWin" sounds like it's installation info ("run this file")

Reply:
I will move this to installation info.